src/keymap.cpp

```diff
@@ -25,7 +25,7 @@ struct OpDesc {
  * This is the list of operations, defining operation, operation name (for
  * keybindings), default key, description, and where it's valid
  */
-static OpDesc opdescs[] = {
+static std::vector<OpDesc> opdescs = {
```

Comment:
I think this can be `const`. Also, `std::array` might communicate the intent better, since it's a fixed-size container.

Reply:
An issue with `std::array` is that you have to specify the size of the array which I kind of dislike.
I've marked it as const which should communicate the same intent.